person.rb

```diff
@@ -23,17 +24,13 @@ def correct_name
     @name
   end
 
+  def create_rental(_rental)
+    Rental.new(date, self, person)
+  end
```

Comment:
- [ ] The previous reviewer asked you to create your create_rental method something like this :arrow_down: cause you are not passing book and date as an argument in the function :blush:

```
def add_rental(book, date)
  Rental.new(date, book, slef)
end
```

Reply:
@HAMMAS-SALEEM Thank you, I missed this accidentally, it's done :heavy_check_mark: